governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ratification Poll for Onboard/Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022
+summary: This is a budget amendment application for Keep3r Network: a decentralized and permissionless keeper network.
+discussion_link: https://forum.makerdao.com/t/mip63c4-sp4-onboard-amend-keeper-network-keep3r-network/14321
+vote_type: Plurality Voting
+categories:
+   - MIPs
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-06-13T16:00:00
+end_date: 2022-06-27T16:00:00
+---
+# Ratification Poll for Onboard/Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, June 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP63c4-SP4 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*During the 1 month period our network was active on Maker’s Keeper Network, we had an expenditure of 1.48 ETH (4800 USD at the time of writing), we expect this cost to be within that range (4,000 - 5,000 USD per month) and thus why we’ve decided to suggest a 4,000 Min Buffer limit. The suggested 20,000 Max Buffer limit is taken into account due to potential gas spikes and the likelihood of more jobs coming to the network within the next few months.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/24779f52630df945595594759e4cb4b757f4c177)
```

Comment:
Use this URL: https://github.com/makerdao/mips/blob/24779f52630df945595594759e4cb4b757f4c177/MIP63/MIP63c4-Subproposals/MIP63c4-SP4.md